Graphs/Planar Graphs/unplanar_graph.concept.yaml

```
@@ -0,0 +1,50 @@
+ru: Непланарный граф
+en: Unplanar graph
+
+definition:
+  ru: Граф, который не может быть изображён на плоскости без пересечения ребер
+  en:
+    A graph, which cannot be represented on a plane without intersection of
+    edges
+  using:
+    concepts: |
+      unplanar_graph
+      graph
+      plane
+      edge
+
+statement:
+  sufficient_condition_unplanar_graph:
+    title:
+      ru: Достаточное условие непланарности графа
+      en: Sufficient condition for graph unplanarity
+    ru:
+      Если граф содержит полный двудольный подграф или полный подграф на 5
+      вершин, то он является непланарным
+    en:
+      If a graph contains a complete bipartite subgraph or a complete subgraph
+      with 5 vertices, then it is non-planar
+    using:
+      concepts: |
+        complete_graph
+        unplanar_graph
+        bipartite_graph
+        vertex
+
+  necessary_condition_unplanar_graph:
+    title:
+      ru: Необходимое условие непланарности графа
+      en: A necessary condition for graph unplanarity
+    ru:
+      Если граф непланарный, то он должен содержать больше 4 вершин, степень
+      которых больше 3, или больше 5 вершин степени больше 2
+    en:
+      If the graph is non-planar, that it must contain more than 4 vertices
```

```suggestion
      If a graph is unplanar, then it must contain more than 4 vertices
```
Кто так вручную переводил коряво?